spec/requests/as_account/video_spec.rb

```diff
@@ -396,7 +396,7 @@
   # @see https://developers.google.com/youtube/v3/docs/videos#fileDetails
   # @see https://developers.google.com/youtube/v3/docs/videos#processingDetails.fileDetailsAvailability
   context 'given one of my own videos' do
-    let(:id) { 'nHz3FnAMH3U' }
+    let(:id) { $account.videos.first.id }
```

Comment:
To match the test code, we better say "given one of my own videos, not *available*" like this. Or find an *available* video and put id of that video

Reply:
Sure. Ideally we'd have some sort of fixture script and we could look up the video by title or something.